### ISSUE TITLE
Need more examples

### ISSUE THREAD

**Issue by fantix** [decentfox/aioh2#6]
* JSON-RPC example
* Anything else?

**Comment by Lekensteyn** [comment]
The examples should probably also integrated with CI, not sure yet how to integrate it (either travis.yml or setup.py)

**Comment by fantix** [comment]
Yeah kinda, like the automated tests. We apologies for not enough resource scheduled on this project so far.
